lib/importers/tp/importer.rb

```diff
@@ -1,15 +1,16 @@
 module Importers
   module TP
     class Importer
-      def initialize(retriever: Retriever, call_record: CallRecord, saver: Saver)
-        @retriever = retriever.new(config: Rails.configuration.x.tp)
+      def initialize(retriever: Retriever, call_record: CallRecord, saver: Saver, config: Rails.configuration.x.tp)
+        @retriever = retriever.new(config: config)
```

Comment:
As discuss, going to leave as is as this allows us this limits the binding to `Rails.configuration.x` as it only has a single point of entry into the code.